src/js/components/organisms/hero-homepage/hero-homepage.ts

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import WithRender from './hero-homepage.vue.html';
+import { HeroHomepageProps } from './hero-homepage.models';
+import { buildSrcSet } from './helpers/build-srcset';
+
+@WithRender
+@Component({
+    name: 'hero-homepage',
+})
+export default class HeroHomepage extends Vue implements HeroHomepageProps {
+    @Prop({ type: String, required: true }) title!: HeroHomepageProps['title'];
+    @Prop({ type: String, required: false }) description?: HeroHomepageProps['description'];
+    @Prop({ type: Object, required: false }) image?: HeroHomepageProps['image'];
+    @Prop({ type: Object, required: false }) ctas?: HeroHomepageProps['ctas'];
+
+    get srcSet(): string {
+        const { url = '' } = this.image || {};
```

Comment:
Please convert to a better readable if statement. More letters are not always worse.